examples/user_guide/15-Streaming_Data.ipynb

```diff
@@ -87,14 +96,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Since ``Pipe`` is completely general the data can be any custom type it provides a completely general mechanism to stream structured or unstructured data. Due to this generality it cannot provide some of the more complex features provided by the ``Buffer`` stream."
+    "This approach of using an element constructor directly does not allow you to use anything other than the default key and value dimensions. One simple workaround this limitation is to use ``functools.partial`` as demonstrated in the **Controlling the backlog section** below.\n",
+    "\n",
+    "Since ``Pipe`` is completely general the data can be any custom type it provides a completely general mechanism to stream structured or unstructured data. Due to this generality it cannot provide some of the more complex features provided by the ``Buffer`` stream that we will now explore."
```

Comment:
I can't parse this sentence.

Reply:
Agreed - I assume you are referring to the last sentence above. I'll rewrite it.

Reply:
Clarified the sentence in 881c70c

Reply:
Second-to-last, starting with Since.

Reply:
I'll edit now with some misc fixes and you can fix up this sentence afterwards if needed.